db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/TimestampOperatorNode.java

```diff
@@ -162,4 +162,10 @@ public ValueNode getClone() throws StandardException {
         return newTS;
     }
 
+	@Override
+	public double getBaseOperationCost() throws StandardException {
+		double localCost = SIMPLE_OP_COST * 2.0;
```

Comment:
why 2

Reply:
Similar to the other place, no particular reason for having 2.0, but just to indicate that it should be at least more expensive than a simple addition, which has cost of `SIMPLE_OP_COST`. The operator may be much more expensive than two addition, but I'm worried that expression evaluation cost accumulates fast, don't want this part to dominate the cost.